### PR TITLE
Feature/dev 123/attendees of event

### PR DESCRIPTION
**Pull request by PAKovalev**
Wrong position of user details dialog was fixed.
Attendees is part of Event item now, not separate widgets. It cased silent issue related to Fly with mode (navigation to user space) also.

[https://momentum.nifty.pm/l/2gSHI0RezHRt](https://momentum.nifty.pm/l/2gSHI0RezHRt)

### CODE REVIEW

File: packages/app/src/core/models/EventItem/EventItem.ts [hunk 1]
```diff
@@ -24,6 +24,7 @@ const EventItem = types
     magicRequest: types.optional(RequestModel, {}),
     fetchAttendeesRequest: types.optional(RequestModel, {}),
     attendees: types.optional(types.array(AttendeeModel), []),
+    attendeesDetails: types.optional(EventAttendees, {}),
```
Comment (dmitry-yudakov) [inline packages/app/src/core/models/EventItem/EventItem.ts:27]: The difference between `attendees` and `attendeesDetails` is not really clear for me and maybe the naming could be improved a bit.

**Review by dmitry-yudakov — APPROVED** [review]

File: packages/app/src/core/models/EventItem/EventItem.ts (hunk 1 above)
Comment (PAKovalev) [reply packages/app/src/core/models/EventItem/EventItem.ts:27]: Yeah, this stuff will be improved during redesign. Attendees model is not clear, I agree.

File: packages/app/src/ui-kit/organisms/EventItem/components/Attendees/components/ProfileView/ProfileView.tsx [hunk 2]
```diff
@@ -0,0 +1,93 @@
+import React, {FC, useMemo} from 'react';
+import {observer} from 'mobx-react-lite';
+import {useTranslation} from 'react-i18next';
+import {capitalize} from 'lodash-es';
+import {PanelLayout, Avatar, Text, IconSvg, Heading} from '@momentum-xyz/ui-kit';
+import {absoluteLink, monthAndYearString, withoutProtocol} from '@momentum-xyz/core';
+
+import {UserSpaceList} from 'ui-kit/molecules';
```
Comment (aminbksi) [inline packages/app/src/ui-kit/organisms/EventItem/components/Attendees/components/ProfileView/ProfileView.tsx:8]: `import {UserSpaceList} from 'ui-kit';`

**Review by aminbksi — APPROVED** [review]

File: packages/app/src/ui-kit/organisms/EventItem/components/Attendees/components/ProfileView/ProfileView.tsx (hunk 2 above)
Comment (PAKovalev) [reply packages/app/src/ui-kit/organisms/EventItem/components/Attendees/components/ProfileView/ProfileView.tsx]: Yeah, auto import. Fixed.

**Review by Daboo1998 — CHANGES_REQUESTED** [review]

File: packages/app/src/core/models/EventItem/models/EventAttendees/EventAttendees.ts [hunk 3]
```diff
@@ -18,7 +22,9 @@ const AttendeesListStore = types
       eventId: types.maybe(types.string),
       eventName: types.maybe(types.string),
       selectedAttendeeId: types.maybe(types.string),
-      attendeeDialog: types.optional(Dialog, {})
+      attendeeDialog: types.optional(Dialog, {}),
+      userSpaceList: types.optional(types.array(UserSpaceDetails), []),
```
[the remark this answers is not in the document]
Comment (Daboo1998) [reply packages/app/src/core/models/EventItem/models/EventAttendees/EventAttendees.ts:26]: Can you pls add a todo then?

[the remark this answers is not in the document]
Comment (PAKovalev) [reply packages/app/src/core/models/EventItem/models/EventAttendees/EventAttendees.ts]: Done

**Pull request merged**